crates/evm/src/tests/call_tests.rs

```diff
@@ -132,50 +134,80 @@ fn call_multiple_test() {
+                                Bytes::from_static(&hex!("000000000000000000000000000000000000000000000000000000000000004000000000000000000000000000000000000000000000000000000000000000a0000000000000000000000000000000000000000000000000000000000000002d4a209fb3a961d8b1f4ec1caa220c6a50b815febc0b689ddf0b9ddfbf99cb74479e41ac0063066369747265611400000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a08000000003b9aca006800000000000000000000000000000000000000000000")),
+                            ).unwrap()
+                        }
+                    ]
                 },
-                gas_used: 26630,
-                log_index_start: 0,
-                l1_diff_size: 255,
-            },
-            Receipt {
-                receipt: reth_primitives::Receipt {
-                    tx_type: reth_primitives::TxType::Eip1559,
-                    success: true,
-                    cumulative_gas_used: 229933,
-                    logs: vec![],
+                    gas_used: 169150,
+                    log_index_start: 1,
+                    l1_diff_size: 1019
                 },
-                gas_used: 26630,
-                log_index_start: 0,
-                l1_diff_size: 255,
-            }
-        ]
+                Receipt {
+                    receipt: reth_primitives::Receipt {
+                        tx_type: reth_primitives::TxType::Eip1559,
+                        success: true,
+                        cumulative_gas_used: 80620,
+                        logs: vec![Log {
+                            address: address!("3100000000000000000000000000000000000001"),
+                            data: LogData::new(
+                                vec![b256!("32eff959e2e8d1609edc4b39ccf75900aa6c1da5719f8432752963fdf008234f")],
+                                Bytes::from_static(&hex!("000000000000000000000000000000000000000000000000000000000000000205050505050505050505050505050505050505050505050505050505050505052a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a2a"))
+                            ).unwrap()
+                            }]
+                        },
+                        gas_used: 80620,
+                        log_index_start: 0,
+                        l1_diff_size: 561
+                },
+                Receipt { receipt: reth_primitives::Receipt { tx_type: reth_primitives::TxType::Eip1559, success: true, cumulative_gas_used: 213563, logs: vec![] }, gas_used: 132943, log_index_start: 1, l1_diff_size: 567 },
+                Receipt { receipt: reth_primitives::Receipt { tx_type: reth_primitives::TxType::Eip1559, success: true, cumulative_gas_used: 257293, logs: vec![] }, gas_used: 43730, log_index_start: 1, l1_diff_size: 255 },
+                Receipt { receipt: reth_primitives::Receipt { tx_type: reth_primitives::TxType::Eip1559, success: true, cumulative_gas_used: 283923, logs: vec![] }, gas_used: 26630, log_index_start: 1, l1_diff_size: 255 },
```

Comment:
I don't like `log_index_start: 1`. Is it okay?

Reply:
looking into it

Reply:
there is no problem, sys txs are in a different block